### ISSUE TITLE
Open data download is broken

### ISSUE THREAD

**Issue by cbourjau** [cbourjau/alice-rs#2]
The upstream URL to the data was changed. See

https://github.com/cernopendata/opendata.cern.ch/issues/2241
https://github.com/cernopendata/opendata.cern.ch/issues/1333

**Comment by cbourjau** [comment]
Fixed by adding self signed cert

**Comment by cbourjau** [comment]
Issue persists!

**Comment by cbourjau** [comment]
closed with ff348f0

**Comment by cbourjau** [comment]
The CERN OpenData portal has changed the format of the files describing the datasets. Datasets are now described in the files:

```
http://opendata.cern.ch/record/1102/files/ALICE_LHC10h_PbPb_ESD_{RUNNUMBER}_file_index.txt
```

Which have a layout like:
```
root://eospublic.cern.ch//eos/opendata/alice/2010/LHC10h/000139038/ESD/0001/AliESDs.root
root://eospublic.cern.ch//eos/opendata/alice/2010/LHC10h/000139038/ESD/0002/AliESDs.root
...
```
The files cannot be downloaded from the link via http. The link has to be modified to:
```
https://eospublichttp.cern.ch/
```
For example:
```
https://eospublichttp.cern.ch/eos/opendata/alice/2010/LHC10h/000139038/ESD/0008/AliESDs.root
```
TLS certificates might also have to be updated as well

**Comment by cbourjau** [comment]
Should be fixed with  afdd3dc
